packages/client/hmi-client/src/components/responsive-matrix/cell-bar-graph.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+	<div class="cell-selected-bar" ref="container"></div>
+</template>
+
+<script lang="ts">
+import { PropType } from 'vue';
+import { select, scaleLinear, scaleBand, axisBottom, axisLeft } from 'd3';
+
+import { SelectedCellValue, SelectedCell } from '@/types/ResponsiveMatrix';
+
+export default {
+	// ---------------------------------------------------------------------------- //
+	// props                                                                        //
+	// ---------------------------------------------------------------------------- //
+
+	// ///////////////////////////////////////////////////////////////////////////////
+
+	props: {
+		update: {
+			type: Number,
+			default() {
+				return 0;
+			}
+		},
+		selectedCell: {
+			type: Array as unknown as PropType<SelectedCell>,
+			default() {
+				return [0, 0, 0, 0];
+			}
+		},
+		dataRowList: {
+			type: null,
+			default() {
+				return [];
+			}
+		},
+		dataColList: {
+			type: null,
+			default() {
+				return [];
+			}
+		},
+		labelRowList: {
+			type: Array as PropType<number[]>,
+			default() {
+				return [];
+			}
+		},
+		labelColList: {
+			type: Array as PropType<number[]>,
+			default() {
+				return [];
+			}
+		},
+		parameters: {
+			type: Array as PropType<string[]>,
+			default() {
+				return [];
+			}
+		},
+		parametersMin: {
+			type: Object,
+			default() {
+				return {};
+			}
+		},
+		parametersMax: {
+			type: Object,
+			default() {
+				return {};
+			}
+		},
+		colorFn: {
+			type: Function,
+			default() {
+				return '#000000';
+			}
+		}
+	},
+
+	// ---------------------------------------------------------------------------- //
+	// data                                                                         //
+	// ---------------------------------------------------------------------------- //
+
+	// ///////////////////////////////////////////////////////////////////////////////
+
+	data() {
+		return {
+			topMargin: 10,
+			bottomMargin: 20,
+			leftMargin: 30,
+			rightMargin: 20,
+			containerBoundingBox: {} as DOMRect,
+			svg: null as any
+		};
```

Comment:
Other than containerBoundingBox it doesn't look like any of the other attributes need to be reactive.

Reply:
yup thats correct